src/biokbase/narrative/tests/test_jobcomm.py

```diff
@@ -3,21 +3,19 @@
 import os
 import itertools
 import re
+import copy
 
-from biokbase.narrative.exception_util import transform_job_exception
-from biokbase.narrative.jobs.jobcomm import exc_to_msg
-import biokbase.narrative.jobs.jobcomm
-import biokbase.narrative.jobs.jobmanager
```

Comment:
the imports had got a bit messed up, so I changed them to import the specific variables / classes / functions that are used